pandas/tests/copy_view/test_indexing.py

```diff
@@ -353,6 +353,8 @@ def test_subset_set_column_with_loc(using_copy_on_write, using_array_manager, dt
     tm.assert_frame_equal(subset, expected)
     if using_copy_on_write or using_array_manager:
         # original parent dataframe is not modified (CoW)
+        # with the enforcement of GH#45333 in 2.0, the original is modified
+        df_orig.loc[1:3, "a"] = np.array([10, 11], dtype="int64")
         tm.assert_frame_equal(df, df_orig)
```

Comment:
This is indeed an example change that should not happen (will also take a look at the code to see what might be the cause)

Reply:
My quick guess is that actually those changes are not needed to have the CoW tests passing (I checked out this branch, and they actually fail because of this change). But I suppose it is the ArrayManager behaviour that changed (it is probably now consistent with the normal block manager behaviour). So you can remove the `or using_array_manager`

Reply:
> I checked out this branch, and they actually fail because of this change

oh, do i need to do an env setting like with PANDAS_DATA_MANAGER?  i had assumed that for this directory it was fixturized so they always ran

Reply:
Yes, it works the same way:

https://github.com/pandas-dev/pandas/blob/42bb8fe60999db433d8f9e1d37117cd2011d291b/.github/workflows/ubuntu.yml#L94-L95